src/extensibility-and-integration/configure-send-emails.md

```diff
@@ -30,6 +30,11 @@ Not sure how to access Service Center? If your environment URL is `outsystems.ex
 
 </div>
 
+## OutSystems SMTP Port Options
+
+OutSystems supports SMTP ports 25, 465, and 587. Consider using secure ports 465 or 587 to avoid potential issues with port 25 throttling. 
```

Comment:
```suggestion
OutSystems supports SMTP ports 25, 465, and 587. However, to avoid potential throttling issues with port 25,  OutSystems recommends using port 587 - the port for encrypted email transmissions using SMTP Secure (SMTPS). Port 465 is also used sometimes for SMTPS. However, this is an outdated implementation and port 587 should be used if possible. 
```